pkgs/servers/gotify/update.sh

```diff
@@ -11,9 +11,9 @@ echo got version $version
 echo \""${version#v}"\" > "$dirname/version.nix"
 printf '%s' $(nix-prefetch-git --quiet --rev ${version} https://github.com/gotify/server | jq .sha256) > $dirname/source-sha.nix
 tput setaf 1
-echo zeroing modSha256 in $dirname/mod-sha.nix
+echo zeroing vendorSha256 in $dirname/vendor-sha.nix
```

Comment:
By the way, you can also use [nix-prefetch](https://github.com/Mic92/nix-update/blob/master/nix_update/update.py#L53) or [nix-update](https://github.com/Mic92/nix-update) itself for tasks like this.

Reply:
Saves a few lines of code.

Reply:
You mean with the `--fetcher` argument?

Reply:
You just need to pass a nix function as string to `nix-prefetch`: https://github.com/Mic92/nix-update/blob/master/nix_update/update.py#L37